### PR TITLE
Updating information on how to pass federations flag to kube-dns

### PR DESCRIPTION
**Pull request by nikhiljindal**
Ref https://github.com/kubernetes/kubernetes/issues/36194

No need to edit the kube-dns-rc now. Just create a configmap with the relevant data.

cc @bowei

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/1788)

### CODE REVIEW

**Comment by bowei** [comment]
do you need to specify that this requires v1.5+?

**Comment by nikhiljindal** [comment]
Thats a very good point @bowei :)

Updated the PR to include instructions for both v1.5 and before.

**Comment by nikhiljindal** [comment]
Also adding @jaredbhatti and @devin-donnelly for docs team review.

**Comment by nikhiljindal** [comment]
@bowei Can you please take another look?

**Comment by nikhiljindal** [comment]
@devin-donnelly Can you please take a look?

**Comment by devin-donnelly** [comment]
I've made some cleanup edits. Please let me know if those work for you. I didn't change any technical content.

File: docs/admin/federation/index.md [hunk 1]
```diff
@@ -14,20 +14,20 @@ This guide explains how to set up cluster federation that lets us control multip
...
 
 ```shell
 $ curl -L https://github.com/kubernetes/kubernetes/releases/download/v1.4.0/kubernetes.tar.gz | tar xvzf -
 $ cd kubernetes
 ```
 
-This guide also assumes that you have an installation of Docker running
-locally, i.e. on the machine where you run the commands described in this
+You must also have a Docker installation running
+locally--meaning on the machine where you run the commands described in this
```
Comment (nikhiljindal) [inline docs/admin/federation/index.md]: should there be spaces around --?

**Comment by nikhiljindal** [comment]
Thanks @devin-donnelly your edits look good!

**Comment by devin-donnelly** [comment]
Cool. Merging now.

**Pull request merged**